profiler/src/ProfilerEngine/Datadog.Profiler.Native/LinkedList.hpp

```diff
@@ -35,7 +22,7 @@ template <class T>
 class LinkedList
 {
 public:
-    LinkedList(pmr::memory_resource* allocator = pmr::get_default_resource()) noexcept :
+    LinkedList(shared::pmr::memory_resource* allocator = shared::pmr::get_default_resource()) noexcept :
```

Comment:
same comment about how to pass a "tag" to track the allocations per provider or other owners

Reply:
same answer :P